libraries/ESP8266WiFi/src/ESP8266WiFiAP.cpp

```diff
@@ -244,21 +247,22 @@ bool ESP8266WiFiAPClass::softAPConfig(IPAddress local_ip, IPAddress gateway, IPA
     dhcp_lease.end_ip.addr = ip.v4();
     DEBUG_WIFI("[APConfig] DHCP IP end: %s\n", ip.toString().c_str());
 
-    if(!dhcpSoftAP.set_dhcps_lease(&dhcp_lease))
+    auto& server = dhcpSoftAP();
+    if(!server.set_dhcps_lease(&dhcp_lease))
     {
         DEBUG_WIFI("[APConfig] wifi_set_ip_info failed!\n");
         ret = false;
     }
 
     // set lease time to 720min --> 12h
-    if(!dhcpSoftAP.set_dhcps_lease_time(720))
+    if(!server.set_dhcps_lease_time(720))
     {

```

Comment:
Lease time is by default 120 seconds, which was a ridiculous default lease time.
Setting it to 720 here is better, but it would be better to make it a default value.

Also these self-`set_`functions always return true.
Would there be a static-constexpr way to avoid this code and messages ?

Reply:
720 is new default, no need to call it then.

true, but that would be a lot of loc changes. redundant isRunning, various sanity checks may as well happen in init? since I am pretty sure I'll also will write options code, keeping changes here at a minimum